libmamba/src/core/output.cpp

```diff
@@ -122,7 +122,11 @@ namespace mamba
                 m_padding = std::vector<int>(n_col, 1);
             }
 
-            std::size_t total_length = std::accumulate(cell_sizes.begin(), cell_sizes.end(), 0);
+            std::size_t total_length = std::accumulate(
+                cell_sizes.begin(),
+                cell_sizes.end(),
+                static_cast<std::size_t>(0)
```

Comment:
Nit: also using `0u` is shorter to read.

Reply:
0u means unsigned, but not necessarily unsigned long long.

Reply:
But the implicit conversion is well defined then